app/src/pages/dataset/RunExperimentButton.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { ReactNode, useCallback, useMemo, useState } from "react";
+
+import {
+  Button,
+  Dialog,
+  DialogContainer,
+  Icon,
+  Icons,
+  Text,
+  View,
+} from "@arizeai/components";
+
+import { PythonBlockWithCopy } from "@phoenix/components/code/PythonBlockWithCopy";
+import { useDatasetContext } from "@phoenix/contexts/DatasetContext";
+
+const INSTALL_PHOENIX_PYTHON = `!pip install arize-phoenix==${window.Config.platformVersion}`;
```

Comment:
```suggestion
const INSTALL_PHOENIX_PYTHON = `pip install arize-phoenix>=${window.Config.platformVersion}`;
```

We may not need this since users will already have Phoenix installed.

Reply:
needed for hosted really.

Reply:
note to self I need to add shell support